docs/community/ecosystem.md

```diff
@@ -0,0 +1,59 @@
+# MXNet's Ecosystem
+
+Community contributions to MXNet have added many new valuable features and functionality to support use cases such as model serving & portability, easy and flexible APIs, and educational material like crash courses and online books. This ecosystem page lists the projects that use MXNet, teach MXNet, or augment MXNet in some way.
+
+
+## Highlighted Project
+
+[![Promo image](https://cdn-images-1.medium.com/max/800/1*PwIMdZM7tpt3rmcyhlk2FQ.png)](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+#### [Keras-MXNet v2.2 released!](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+
+## Contents
+
+* [Learning MXNet and other Educational Resources](#learning-mxnet-and-other-educational-resources)
+* [New MXNet APIs](#new-mxnet-apis)
+* [Toolkits to Extend MXNet](#toolkits-to-extend-mxnet)
+* [Model Serving](#model-serving)
+* [Model Zoos](#model-zoos)
+* [MXNet as a Backend](#mxnet-as-a-backend)
+* [Contributions](#contributions)
+
+## Learning MXNet and other Educational Resources
+
+* [Gluon 60 Minute Crash Course](https://gluon-crash-course.mxnet.io/) - deep learning practitioners can learn Gluon quickly with these six 10-minute tutorials.
```

Comment:
Add link to supporting YouTube series? https://www.youtube.com/playlist?list=PLkEvNnRk8uVmVKRDgznk3o3LxmjFRaW7s